eZ/Publish/FieldType/Tweet/NameableField.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * File containing the NameableField class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet;
+
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+use eZ\Publish\SPI\FieldType\Nameable;
+use eZ\Publish\SPI\FieldType\Value;
+
+class NameableField implements Nameable
```

Comment:
yes, lets keep it simple.



Note: that the deprecation on `getName` might be something to consider dropping, Nameable is optional in my mind, it's for the more advance use cases.